### PR TITLE
Add `no-extra-semi` to match what's in sonarcloud

### PR DESCRIPTION
**Pull request by chmelevskij**
Trying to catch it earlier
Screenshot 2022-08-19 at 22 18 11

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
